lib/src/_shared/BasePicker.jsx

```diff
@@ -31,7 +31,7 @@ export const BasePickerHoc = compose(
     handleSetTodayDate: ({ changeDate, utils }) => () => changeDate(utils.date()),
     handleTextFieldChange: ({ changeDate, onChange }) => (date) => {
       if (date === null) {
-        this.handleClear();
+        onChange(null);
```

Comment:
I don't know is there opened issues, but this shouldn't work according spec. `this` is arrow function should reference to the nearest scope which is module in this case. And according spec module doesn't have `this` so it will be `undefined`. Rollup is able to detect such things. It's weird that this case ever worked for you.